modules/apps/segments/segments-web/test/js/components/criteria_builder/ContributorsBuilder.es.js

```diff
@@ -180,6 +182,17 @@ const propertyGroups = [
 	},
 ];
 
+const scopeName = 'Liferay DXP';
```

Comment:
If you only use this constant in the scope card test maybe you can move it there.

Reply:
Changed! thank u